beetsplug/kodinfo.py

```diff
@@ -0,0 +1,43 @@
+
+# -*- coding: utf-8 -*-
+# This file is part of beets.
+# Copyright 2017, Sergio Soto.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+"""Genertes artist.nfo and album.nfo after the import process
+These are used by Kodi and other platforms to save the MBID
+and other details.
+For singletons, it generates one .nfo for each track.
+"""
+
+from beets.plugins import BeetsPlugin
+
+
+class KodiNfo(BeetsPlugin):
+    def __init__(self):
+        super(KodiNfo, self).__init__()
+        self.register_listener('album_imported', self.make_album_nfo)
+        self.register_listener('item_imported', self.make_item_nfo)
+
+    def make_album_nfo(self, lib, album):
+        linkalbum = 'https://musicbrainz.org/release/{0}'
+        linkartist = 'https://musicbrainz.org/artist/{0}'
```

Comment:
Since these never change, it's good style to write them as *constants*. You can hoist them to the top of the module and spell them in all caps, like this:

```py
LINK_ALBUM = 'https://musicbrainz.org/release/{0}'
LINK_ARTIST = 'https://musicbrainz.org/artist/{0}'
```